build/build-node-agent-image

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+BOOTKUBE_ROOT=${BOOTKUBE_ROOT:-$(git rev-parse --show-toplevel)}
+
+DOCKER_REPO=${DOCKER_REPO:-quay.io/derek_parker/node-agent}
```

Comment:
If all these build-image scripts are essentially the same (minus the tool name) we might want to make a generic script that takes the name as a parameter.